src/script/main/app.ts

```diff
@@ -492,7 +492,6 @@ export class App {
       telemetry.timeStep(AppInitTimingsStep.APP_LOADED);
 
       telemetry.report();
-      amplify.publish(WebAppEvents.LIFECYCLE.LOADED);
```

Comment:
are you sure about this change?

Reply:
I think it's still needed for the desktop app. Please check